gradle/publishing.gradle

```
@@ -76,7 +76,7 @@ publishing {
                     }
                 }
 
-                project.configurations.api.dependencies.each { addDependency(it, 'compile') }
+                project.configurations.api.dependencies.findAll { it.name != "docker-java-bom" }.each { addDependency(it, 'compile') }
```

I couldn't find a quick way to fix this, so I had to add this ugly workaround :(

---

Shouldn't this be `implenentation` ? 
```suggestion
                project.configurations.api.dependencies.findAll { it.name != "docker-java-bom" }.each { addDependency(it, 'implementation') }
```

---

there is no "implementation" in Maven POMs, you confused it with super innovative Gradle 😂 Also, even so, it would be `api` and not `implementation` :D